.github/workflows/CI.yml

```diff
@@ -38,9 +39,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: "12.x"
+          cache: yarn
```

Comment:
There's already some caching logic in this file. I guess the old logic can be removed?

Reply:
Yep, I think it can be removed.